sdk/containerregistry/azure-containerregistry/azure/containerregistry/_models.py

```diff
@@ -312,6 +313,60 @@ def repository_name(self):
         return self._repository_name
 
 
+class DownloadBlobResult(object):
+    """The result from downloading a blob from the registry.
+
+    :ivar data: The blob content.
+    :vartype data: IO
+    :ivar str digest: The blob's digest, calculated by the registry.
+    """
+
+    def __init__(self, **kwargs):
+        self._data = kwargs.get("data")
+        self._digest = kwargs.get("digest")
+
+    @property
+    def data(self):
+        # type: () -> IO
+        return self._data
+
+    @property
```

Comment:
Why have we wrapped everything in `@property`?

Reply:
Have updated to attributes.